scripts/us_eia/opendata/process/common.py

```diff
@@ -211,6 +211,12 @@ def process(in_json, out_csv, out_sv_mcf, out_tmcf, extract_place_statvar_fn,
                         counters['error_empty_values'] += 1
                         continue
 
+                    try:
+                        _ = float(v)
```

Comment:
Some other values I've seen:

(INTL):
-- = Not applicable
- = No data reported
(s) = Value is too small for the number of decimal places shown
NA = Not available
W = Data withheld to avoid disclosure
* = Conversion Factor Unavailable
se = EIA estimates based on time series analysis
st = EIA forecasts (Short-Term Energy Outlook)

(AEO):
- - = Not applicable.

(COAL):
W = Withdrawn

In fact, I'm coming across "null" in the json - see series_id=COAL.COST.CA-94.A

Could we make a note of these here, so we can revisit?

Reply:
Awesome find! I've added these. For my future reference, where did you find these?

I also fixed a bug where 0.0 was skipped. Added additional test cases too.

Reply:
I found these in the footnotes of some of the charts (in the data browser, not the opendata query browser), e.g. click on the [notes and sources link here](https://www.eia.gov/international/data/world/other-statistics/emissions-by-fuel?pd=40&p=0000000000000000000000000000000000000000000000000000000b0001&u=0&f=A&v=mapbubble&a=-&i=none&vo=value&&t=C&g=00000000000000000000000000000000000000000000000001&l=249-ruvvvvvfvtvnvv1vrvvvvfvvvvvvfvvvou20evvvvvvvvvvnvvvs0008&s=-662688000000&e=1514764800000).

The other tabs are also incredibly rich, though it's not as consistent across datasets.